swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
+DESCRIPTION_DICT = {
+    0: "Did you even check?",
+    1: "Possibly good for parts, possibly.",
+    2: "A good holiday gift for your least favorite relative. ",
+    3: "Not bad...",
+    4: "Worth the price of admission.",
+    5: "A hidden treasure for Antiques Roadshow.",
+}
+
+
 class Item:
-    pass
\ No newline at end of file
+
+    def __init__(self, category = "", condition=0):
+        """
```

Comment:
Nothing wrong with some comments! This code is considered "self-documenting" since it's extremely clear what it's doing. Committing comments is usually most helpful when it's adding clarity to some complex logic, explaining usage details, or leaving a note for the next person who might come along and want some context (amongst other reasons). Just something to think about!